### PR TITLE
Pull Request for Milestone #2 - TicTacToe Game

### PR DESCRIPTION
**Pull request by tadeuasarro**
###### For this milestone we have created an interactive user's display and the game flow.

### CODE REVIEW

File: bin/main.rb [hunk 1]
```diff
@@ -1,2 +1,68 @@
...
+
+  print " #{player1} Enter the position which you want to place your #{tokens[turn]}: "
+  move = gets.chomp.to_i
+  next if move.zero? || move > 9 || move < 1
+
+  board[move - 1] = tokens[turn]
+  turn.zero? ? turn=1 : turn=0
+  board.all?(String) ? end_game = true : nil
+  puts move
+  break unless end_game != true
```
Comment (bertil291utn) [inline bin/main.rb]: ## Contidional to end game 
- Change to 
```ruby
break if game_ends
```
Where game_ends is a boolean variable

**Review by bertil291utn — CHANGES_REQUESTED** [review]
# **Status:** Changes requested 🟠 
<p align="center"> image <br><strong>Not yet</strong> </p>

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

### Your work is amazing you master it GitHub Flow. 

However, in order to accomplish all the requirements, you required to do these changes, listed below:

- [x] Add the linter directory `.github/workflows/linters.yml`, follwing this [guideline](https://github.com/microverseinc/linters-config/tree/master/ruby)

- Work on these methods
- [x] Please add `[REQUIRED]` game informs player if the selected move is invalid
- [x] `[REQUIRED]` game displays board after the player move
* [ ] `[REQUIRED]` game informs player if the selected move is a winning move
* [ ] `[REQUIRED]` game informs player if the selected move is a draw move

- [x] On your first review you must have at least 10 commit messages, don't forget next time

### Summary
- So far, your code is good and clean, and remember for now your code should not work, just think what steps you need to solve play the game
-  Don't forget to work with inline comments below this comment box.

***

Guys work in the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

File: bin/main.rb [hunk 2]
```diff
@@ -1,2 +1,76 @@
...
+    puts 'Out of range select another position'
+    next
+  elsif board[move - 1].is_a? String
+    puts 'Position taken!!!! Select another one'
+    next
+  end
+
+  board[move - 1] = tokens[turn]
+  turn = turn.zero? ? 1 : 0
+  board.all?(String) ? end_game = true : nil
```
Comment (expjazz) [inline bin/main.rb]: - [x] I do not see you checking for winners. Add something related to this (remember that still does not have to fully work yet), and you also need to inform the players if a move is a winning move.

**Review by expjazz — CHANGES_REQUESTED** [review]
## Status: Changes Required :recycle: 

Hello team :+1: 

You did a great job here. You did a very creative way to validate the moves :ok_hand: 

However, I did not see you checking for a winner. Kindly add that (for more details check the inline comment below).

#### It was not required to do the readme up to this point, but you still did it. Awesome :clap: 
### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Review by kubilaycaglayan — APPROVED** [review]
Hello Team 👋 
Status: Project Approved 🟢 

Good job on making all the changes!

Good luck with the rest of the project...
With all my respect... 🙏

**Pull request merged**